language/move-prover/src/spec_translator.rs

```diff
@@ -703,19 +725,24 @@ impl<'env> SpecTranslator<'env> {
     }
 
     fn translate_local_var(&self, node_id: NodeId, name: Symbol) {
-        self.auto_dref(node_id, || {
+        let mut ty = &self.module_env().get_node_type(node_id);
```

Comment:
In this code block, I am trying to budget for translation of variables in spec functions, in lambdas, and local variables in Move functions.

Reply:
Please add a comment below that you override `ty` based on transformation results in func_target.

Reply:
Done.